index.html

```diff
@@ -43,6 +43,7 @@
         <li><a href="/user/build-timeouts">My builds are timing out, what can I do?</a></li>
         <li><a href="/user/repository-providers">What repository providers can I use?</a></li>
         <li><a href="/user/pricing/">Why can't I find pricing information?</a></li>
+        <li><a href="/user/ci-environment-trusty-beta/">Can I use Ubuntu Trusty for my builds?"</a><li>
```

Comment:
Not sure, but it is there, and changing that isn't simple or nessicary right now.

As we have `/user/ci-environment/`, I would suggest we go for `/user/ci-environment/trusty` maybe?